event_listener/dfusion_db/event_pusher.py

```diff
@@ -69,11 +78,34 @@ def update_accounts(event: Dict[str, Union[int, str, str, int]]):
         db.accounts.insert_one(new_account_record)
 
     elif transition_type == 1:  # Withdraw
-        pass
+
+        requested_withdraws = db.withdraws.find({'slot': event['slot']})
+
+        for deposit in requested_withdraws:
```

Comment:
bit: withdraw

Reply:
Are you aware of the "Suggestion" button here? I would be way easier for me to change lines like this directly in the browser.

Reply:
Ah, yes, but I will need to do it on several lines.

Reply:
Fixed!